lib/enhance-assert.js

```diff
@@ -1,69 +1,82 @@
 'use strict';
 
-module.exports = enhanceAssert;
-module.exports.formatter = formatter;
+const dotProp = require('dot-prop');
 
-module.exports.PATTERNS = [
+const PATTERNS = [
 	't.truthy(value, [message])',
 	't.falsy(value, [message])',
 	't.true(value, [message])',
 	't.false(value, [message])',
-	't.is(value, expected, [message])',
-	't.not(value, expected, [message])',
-	't.deepEqual(value, expected, [message])',
-	't.notDeepEqual(value, expected, [message])',
-	't.regex(contents, regex, [message])',
-	't.notRegex(contents, regex, [message])',
```

Comment:
You'll have to update https://github.com/avajs/babel-preset-transform-test-files/blob/master/espower-patterns.json as well. Which is a breaking change 😝